client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-large-preview.tsx

```diff
@@ -70,31 +68,35 @@ const PatternLargePreview = ( {
 		goToSelectHeaderPattern();
 	};
 
+	const getTitle = () => {
+		if ( ! shouldShowSelectPatternHint ) {
+			return translate( 'Welcome to your blank canvas.' );
+		}
+
+		return translate( 'Ready to start designing?' );
+	};
+
 	const getDescription = () => {
 		if ( ! shouldShowSelectPatternHint ) {
 			return translate( "It's time to get creative. Add your first pattern to get started." );
 		}
 
-		const options = {
-			components: {
-				link: (
-					// eslint-disable-next-line jsx-a11y/anchor-is-valid
-					<a href="#" target="_blank" rel="noopener noreferrer" onClick={ handleAddHeaderClick } />
-				),
-			},
-		};
-
-		return hasEnTranslation(
-			'You can view your color and font selections after you select a pattern. Get started by {{link}}adding a header pattern{{/link}}'
-		)
-			? translate(
-					'You can view your color and font selections after you select a pattern. Get started by {{link}}adding a header pattern{{/link}}',
-					options
-			  )
-			: translate(
-					'You can view your color and font selections after you select a pattern, get started by {{link}}adding a header pattern{{/link}}',
-					options
-			  );
+		return translate(
+			'You can view your color and font selections after you select a pattern. Get started by {{link}}adding a header pattern{{/link}}',
```

Comment:
Let's add a period at the end of this sentence: `Get started by {{link}}adding a header pattern{{/link}}.`

Reply:
Oops, nice catch!